compass/compass.js

```diff
@@ -0,0 +1,53 @@
+﻿API.onUpdate.connect(function (sender) {
+
+    if (0.3 < API.getGameplayCamDir().X && 0.3 < API.getGameplayCamDir().Y) {
+
+        API.drawText("NE", 350, 1000, 1, 255, 255, 255, 255, 2, 1, false, true, 0);

```

Comment:
You can also store the string in a temporary variable instead of duplicating the same `drawText` call every time.